sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -936,15 +1006,43 @@ def close(self):
       pass
 
 
+class WorkerInfo(object):
```

Comment:
I think this class can go away, its parts merged into either WorkerHandler and ControlConnection (above).